### ISSUE TITLE
Show number of occurrences of highlighted token in status bar

### ISSUE THREAD

**Issue by EuropeanDeveloper** [notepad-plus-plus/notepad-plus-plus#11312]
When a token is highlighted it would be helpful to show number of its occurrences in the status bar. I'm aware that the number of token occurrences can be found out via Find window (`Ctrl+F` > `Count`) but this feature could be useful especially on smaller files where you want to count quickly how many times is token present in a file.

**Comment by alankilborn** [comment]
Just pointing out that _Search_ > _Mark..._ already does this nicely:

![image](https://user-images.githubusercontent.com/77065706/156170128-4feea822-7991-45ea-b071-eb62a886a959.png)

**Comment by EuropeanDeveloper** [comment]
Yes, I'm aware of that. As I already pointed out, you can do this also via `Find` > `Count`:

![screenshot](https://user-images.githubusercontent.com/52759436/156182810-83a2672f-6e50-412f-bf21-e762ee772575.png)

But this takes three steps:
1. Selection of a token.
2. Opening `Find` window.
3. Clicking on `Count`.

The proposed feature would do this much quicker, just in one step (by selecting the token). In status bar could be shown, e.g.:

![statusbar](https://user-images.githubusercontent.com/52759436/156184175-6ca1c34d-d53f-4ab5-9c6e-cecd12f69863.png)

**Comment by dfs-** [comment]
I see your point when for example you want to know if there are any references to a specific word/function inside the same file, but I would suspect this to be a performance killer as it must perform the search with any currently selected text. 

As soon as we are talking about huge or giant files it would certainly hinder the workflow. 

However isn’t it possible to create something like this in the PythonScript plug-in which you could then execute using a self-defined hotkey?

**Comment by alankilborn** [comment]
>However isn’t it possible to create something like this in the PythonScript plug-in which you could then execute using a self-defined hotkey?

Yes.

    t = editor.getSelText()
    if len(t):
        matches = []
        editor.search(t, lambda m: matches.append(m.span(0)))
        notepad.menuCommand(MENUCOMMAND.SEARCH_MARKALLEXT1)
        notepad.messageBox('Occurrences of selected text: {}'.format(len(matches)), '')
